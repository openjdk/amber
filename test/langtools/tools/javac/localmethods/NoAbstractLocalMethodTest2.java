/*
 * @test /nodynamiccopyright/
 * @bug 8233038
 * @summary Tests need to be written for local methods feature
 * @compile/fail/ref=NoAbstractLocalMethodTest2.out -XDrawDiagnostics NoAbstractLocalMethodTest2.java
 */

public class NoAbstractLocalMethodTest2 {
    public static void main(String [] args) {
        void yyy (int y);
    }
}
