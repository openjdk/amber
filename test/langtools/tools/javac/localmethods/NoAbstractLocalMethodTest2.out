NoAbstractLocalMethodTest2.java:10:14: compiler.err.missing.meth.body.or.decl.abstract
1 error
